common/pom.xml

```diff
@@ -42,6 +42,13 @@
         </keycloak.osgi.import>
     </properties>
     <dependencies>
+
+        <dependency>
+            <groupId>org.wildfly.security</groupId>
+            <artifactId>wildfly-elytron</artifactId>
+            <version>2.0.0.Beta1</version>
```

Comment:
It will be good to remove `version` line, which means that it will use same version used in the `pom.xml`, which is currently `1.18.3.Final` . AFAIK we need to use this version due the Wildfly uses it.

Could you remove the `version` line? If there is no big change in the elytron API between 1.18.3.Final and 2.0.0..Beta1, your code will still work fine though (hopefully that is the case).